Quaternion/Quaternion.py

```diff
@@ -7,7 +7,7 @@
    - class methods to multiply and divide quaternions
 
 :Copyright: Smithsonian Astrophysical Observatory (2010)
-:Author: Jean Connelly (jconnelly@cfa.harvard.edu)
+:Authors: Tom Aldcroft (aldcroft@cfa.harvard.edu), Jean Connelly (jconnelly@cfa.harvard.edu)
```

Comment:
@javierggt Now needs to be in here too!

Reply:
done, but the line is now too long. I am not used to this docstring format, so I do not know how off the top of my head how to make it in multiple lines.

Reply:
Looks like you can do this:
```
:Authors: - Me
          - Myself
          - I
```